manifests/c/CryptoPro/Chromium-Gost/108.0.5359.94/CryptoPro.Chromium-Gost.locale.ru-RU.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: CryptoPro.Chromium-Gost
+PackageVersion: 108.0.5359.94
+PackageLocale: ru-RU
+Publisher: CryptoPro

```

Comment:
```suggestion
Publisher: The Chromium-Gost Authors
```

![image](https://user-images.githubusercontent.com/88161975/206768884-6a50d1b9-27c1-4fca-a783-e5b3d856cd23.png)
This must match Control Panel/Apps&Features